recipes/bwa-mem2/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+LIBS="${LDFLAGS}" make CC="${CC}" CXX="${CXX}" multi
```

Comment:
Explanation for the curious:

If we put `LIBS` into the environment (rather than a `make LIBS="${LDFLAGS}"` parameter), the `LIBS+=` in the (patched) `Makefile` will include everything we want.
(Same with the `CPPFLAGS` which are set by the compiler activation scripts: with `CPPFLAGS+=` in the `Makefile` we still have our compiler activation script include paths intact.)

For `CC` and `CXX` we really only want the values we pass on and hence give them via `X=Y` parameters to the `make` call.